runtime/include/bpftime_config.hpp

```diff
@@ -4,6 +4,11 @@
 #include <cstdlib>
 #include <string>
 
+#ifndef DEFAULT_LOGGER_OUTPUT_PATH
+#define DEFAULT_LOGGER_OUTPUT_PATH "~/.bpftime/runtime.log"
```

Comment:
Ok, I can fix that in next pr

Reply:
The problem is that it might be input by the cmake... is it better to 
```c
#ifndef DEFAULT_LOGGER_OUTPUT_PATH
const char* default_log_path =  "~/.bpftime/runtime.log"
#else    
const char* default_log_path =  DEFAULT_LOGGER_OUTPUT_PATH
#endif
```
I think the old one is  a straightforward and clean way to ensure that a default value is set if none is provided.